app/main.py

```diff
@@ -2,4 +2,58 @@ class Cargo:
     def __init__(self, weight: int) -> None:
         self.weight = weight
 
-# write your code here
+
+class BaseRobot:
+    def __init__(self, name: str, weight: int, coords: list = None) -> None:
+        self.weight = weight
+        self.name = name
+        self.coords = [0, 0] if coords is None else coords
```

Comment:
I think it's better that way
```suggestion
        self.coords = coords or [0, 0]
```